cypress/fixtures/doi.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://datacite.org/schema/kernel-4" xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4/metadata.xsd">
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://datacite.org/schema/kernel-4" xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4.5/metadata.xsd">
```

Comment:
Good observation.  I just checked and it looked like Kuda has made them consistent.

Reply:
Good observation.  I just checked and it looked like Kuda has made them consistent.  However, it seems like it doesn't matter for fixtures.  My question is why in the schemaLocation do we have 2 urls?  As in:

```xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4/metadata.xsd"```

Is one for documentation and one for the actual definition?

Reply:
Good question. What you say makes sense about linking to documentation.

I am wondering about https://datacite.org/schema/kernel-4. This redirects to https://schema.datacite.org/meta/kernel-4.4/. Where is this set up? It would be better to redirect to https://schema.datacite.org/meta/kernel-4/.